teuthology/dispatcher/supervisor.py

```diff
@@ -275,7 +275,7 @@ def unlock_targets(job_config):
 
 
 def run_with_watchdog(process, job_config):
-    job_start_time = datetime.utcnow()
+    job_start_time = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
looks gut, I know it must be the same to use shorter form `datetime.UTC`, but why did you prefer longer variant, for the reference:
```
DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
```

Reply:
Because they didn't make it available as `datetime.UTC` until 3.11 🙃
https://docs.python.org/3/library/datetime.html#datetime.UTC